src/classes/BDI_DataImportDeleteBTN_TEST.cls

```diff
@@ -41,6 +41,15 @@ class BDI_DataImportDeleteBTN_TEST {
     /** @description Error message returned when user does not have Delete permission on the SObject */
     static String INSUFFICIENT_ACCESS_ERROR_MSG = 'INSUFFICIENT_ACCESS_OR_READONLY';
 
+    /** @description Identifier for when Salesforce classic is displayed for the user */
+    static final String UI_CLASSIC_THEME = 'Theme3';
+
+    /** @description Identifier for when Lightning Experience is displayed for the user */
+    static final String UI_LIGHTNING_THEME = 'Theme4d';
+
+    /** @description Data Import SObject API name */
+    static final String DATA_IMPORT_API_NAME = 'DataImport__c';
```

Comment:
Let's use the token instead of introducing this string.  See other comment above and let me know if you have any questions!

Reply:
fixed!